playbooks/openstack/openshift-cluster/launch.yml

```diff
@@ -128,6 +128,6 @@
       - parsed_outputs.node_floating_ips
       - parsed_outputs.infra_floating_ips
 
-- include: update.yml
+- include: setup.yml
```

Comment:
@akram In the other playbooks (aws in particular), launch calls the 'update.yml' playbook. It seems that your setup.yml playbook is similar to a normal update.yml playbook. Mind renaming it to update.yml so that we can keep consistency?